docs/DevGuide/BuildSystem.md

```diff
@@ -311,14 +311,7 @@ cmake -D FLAG1=OPT1 ... -D FLAGN=OPTN <SPECTRE_ROOT>
     (default is `ON`)
 - USE_GIT_HOOKS
   - Use git hooks to perform certain sanity checks so that small goofs are
-    caught before getting to CI. Most situations where the hooks can cause
-    problems are handled automatically. The exception is when the build
```

Comment:
I think you should keep more of this comment up to "automatically."

Reply:
🤷‍♂️ I thought without the Py2 bug this comment doesn't really say much other than "hopefully the git hooks are not buggy" and "if you know what's good for you, you better use them". Neither is particularly useful. I'd rather remove the comment, but I don't feel strongly about it.

Reply:
[optional] Might be worth keeping the note that CI does the same checks, so you still have to satisfy them even if you turn this off.

Reply:
Reworded to:

> Use git hooks to perform some sanity checks so that small goofs are caught before they are committed. These checks are particularly useful because they also run automatically on \ref github_actions_guide "CI" and must pass before pull requests are merged.